### DISCUSSION TITLE
tvtime - sync

### DISCUSSION THREAD

**Discussion by luciofsl** [pymedusa/Medusa#7886] [Ideas & Feature requests]
[tvtime](https://www.tvtime.com/) is a platform that allow you to mark your episode watched. 
It would be nice to medusa sync this data from tvtime, and allow us to have this information in series page. 
This info is very useful to allow us manage the storage disk with more secure, for example you can filter by episodes seen and delete them.

**Comment by OmgImAlexis** [comment]
It makes more sense to have something like this integrated with Kodi/Plex than it does Medusa.

**Comment by luciofsl** [comment]
> It makes more sense to have something like this integrated with Kodi/Plex than it does Medusa.

I agree with you in case that you just watch episodes from one place, even so with this info synced with kodi/plex it would be difficult to manage all the episodes that you can delete in storage.

I watch series from some kodi players at home and others from my tablet that dont have the same library.

**Comment by p0psicles** [comment]
What does tvtime do that trakt doesn't. Trakt also has a Kodi add-on. And keeps track of episodes watched status. I would be more into extending our trakt functionality

**Comment by enrique131** [comment]
> What does tvtime do that trakt doesn't. Trakt also has a Kodi add-on. And keeps track of episodes watched status. I would be more into extending our trakt functionality

I use both on Kodi, and I have the paid version of trakt.tv. And i can say that tvtime doesn't fail that often as traktv does.

**Comment by OmgImAlexis** [comment]
Not sure what you’re taking about. Used trakt for years myself and never had it “fail”. 🤔

**Comment by enrique131** [comment]
> Not sure what you’re taking about. Used trakt for years myself and never had it “fail”. 🤔

Sometimes it skips shows, or even marks wrong ones. The app takes a lot to sync(seriesGuide). Tvtime is almost instantly updated... but some shows i follow are not in one of them... and sometimes in the other. 

Not an issue as both are working mostly automatically. But they are not perfect.
